MetaMask/metamask-extension
# Token allowance i18n interpolation system

## Explanation

Fixed setSpendingCap, reviewSpendingCap and revokeSpendingCap locale string using the i18n interpolation system.

## More Information

* Fixes #16234
* Blocked by: #16157

## Manual Testing Steps
**BEFORE TESTING** - in your .metamaskrc file, set TOKEN_ALLOWANCE_IMPROVEMENTS=1 and then start the app

1. Open the Test Dapp: https://metamask.github.io/test-dapp/
2. Under the section Send Tokens click on CREATE TOKEN
3. Wait for the token to be created and then, under the same section, click on APPROVE TOKENS or APPROVE TOKENS WITHOUT GAS in order to see the newly created screen

To see how the screen looks like when the enhanced gas fee is enabled, go to Settings (Experimental tab) and enable it (Enable enhanced gas fee UI toggle)
